movie-list.json

```diff
@@ -0,0 +1,407 @@
+[
+    {
+        "popularity": 439.949,
+        "vote_count": 2621,
+        "video": false,
+        "poster_path": "/xBHvZcjRiWyobQ9kxBhO6B2dtRI.jpg",
+        "id": 419704,
+        "adult": false,
+        "backdrop_path": "/5BwqwxMEjeFtdknRV792Svo0K1v.jpg",
+        "original_language": "en",
+        "original_title": "Ad Astra",
+        "genre_ids": [
```

Comment:
Skąd wiesz jaka wartość jest czym?
Nie mówię, żeby to wywalić ale jak potem wykorzystasz te id?

Reply:
dodałam zmienną genre, która przypisuje id do gatunku filmu